composer.json

```diff
@@ -29,6 +29,7 @@
     }
     , "require-dev": {
         "phpunit/phpunit": "4.1.*"
+        , "cakephp/cakephp-codesniffer": "2.*"
```

Comment:
This is a big eye sore for me. Can you please switch to having comma at end of statement for all foc projects :smile: